package.json

```diff
@@ -12,10 +12,11 @@
 		"node": ">=6.1"
 	},
 	"scripts": {
-		"test": "npm run lint && npm run compile-typescript && npm run test-unit",
+		"benchmarks": "node ./benchmarks/index.js",
+		"test": "npm run lint && npm run compile-typescript && npm run test-unit && npm run benchmarks",
```

Comment:
not sure I wanna run the benchmarks as part of `test`

Reply:
Happy to remove it. We do not fail the build, just output report. What would you prefer? Remove entirely? Move to `prepublishOnly`? Something else?